testing/src/cypress/custom-command/institution/common-helper-functions.cy.ts

```diff
@@ -64,7 +64,13 @@ export default class InstitutionHelperActions {
       Cypress.env("ENDPOINTS").CREATE_INSTITUTION_LOCATION
     }`;
   }
-
+  
+  getApiForDesignationAgreement() {
+    return `${this.getBaseUrlForTestEnv()}${
+      Cypress.env("ENDPOINTS").DESIGNATION_AGREEMENT
```

Comment:
`DESIGNATION_AGREEMENT` is not something that will change per environment, right? Same for all API endpoints. Are we still pending these modifications as per the last PR reviews?

Reply:
Thats correct, the dynamic environment ingestion is not done yet. Will update these attributes with that change